src/errors/InvalidMaxOfSliceError.ts

```diff
@@ -0,0 +1,10 @@
+import { Annotated } from './Annotated';
+
+export class InvalidMaxOfSliceError extends Error implements Annotated {
+  specReferences = ['http://www.hl7.org/fhiR/profiling.html#slice-cardinality'];
+  constructor(public sliceMax: string, public sliceName: string, public slicedElementMax: string) {
+    super(
+      `No individual slice may have max >= max of sliced element, but max ${sliceMax} of slice ${sliceName} > max of sliced element ${slicedElementMax}.`
```

Comment:
I think you mean `>`, _not_ `>=`.  It's perfectly fine for the `max`es to be `=`.

Also, the wording is a little inconsistent at the end of the sentence.  It should either be:
```
but max ${sliceMax} of slice ${sliceName} > max ${slicedElementMax} of sliced element.
```
or
```
but max of slice ${sliceName} (${sliceMax}) > max of sliced element (${slicedElementMax}).
```

I _think_ I prefer the latter, but it's not a strong opinion.